src/calculateRentalCost.js

```diff
@@ -5,6 +5,21 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG__TERM = 7;
```

Comment:
The variable name `LONG__TERM` contains double underscores, which is unconventional. Consider using a single underscore, like `LONG_TERM`, for better readability.